pyproject.toml

```diff
@@ -8,8 +8,16 @@ requires = [
 
 build-backend = 'setuptools.build_meta'
 
-[tool.isort]
+[tool.ruff]
+line-length = 95
+select = ["F", "E", "W", "I001"]
+
+[tool.ruff.isort]
+force-single-line = true
+known-first-party = ["pystack"]
+known-third-party=["rich", "elftools", "pytest"]
 
+[tool.isort]
 force_single_line = true
 multi_line_output = 3
 include_trailing_comma = true
```

Comment:
Do we feel strongly about keeping the import style the same as it is? I personally don't mind either way, as long as we have a consistent and enforceable style in the long run. Maybe we could remove `isort` altogether and use `ruff`'s convention instead?

It looks like these three options were introduced by Pablo when the project was started and never changed since then.

Reply:
That’s right. The Makefile (before the cleanup) had flake8 running on Python files and isort running on both Python and Cython files.